SharedFrameworkValidation.cmd

```diff
@@ -1,9 +1,14 @@
 @if not defined _echo @echo off
```

Comment:
yeah I would prefer these not be in the root for this specialized scenario. Next time you are touching this it would be good if you can move it. Thanks.